src/renderer/captureTool/js/common-event-bindings.js

```diff
@@ -0,0 +1,86 @@
+"use strict";
+
+/**
+ * GPII Binder Markup Events Grade
```

Comment:
Add some notes here to explain i) where this code was cribbed from, ii) where it is currently in progress to be incorporated into gpii-binder. In practice this was only ever scratch-quality code and if it ever did get upgraded it would need to be given more meaningful names than "fluid.expandCompoundArg"

Reply:
Added description and links to jira ticket and current github work.